app/utils/decorator_utils.py

```diff
@@ -0,0 +1,12 @@
+from app.database.models.user import UserModel
+
+
+def check_mail_confirmation(func_to_validate):
+
+    def to_validate_user(user_id, data):
+        user = UserModel.find_by_id(user_id)
+        if user.is_email_verified:
+            return func_to_validate(user_id, data)
```

Comment:
Renaming a function from a generic variable name to something like func_to_something isn't very helpful either. Please take into account newcomers like yourself would read this code at some point and begin contributing. I don't want this to be tough for them.

Reply:
would you like to suggest some name?

Reply:
@ramitsawhney27

Reply:
It'd be more helpful if you'd read up on code readability and then handle this. That way you'll be able to do this on your own in future PRs and you'll get to learn.